### PR TITLE
Pop up window and project cards

### PR DESCRIPTION
**Pull request by tafaramafemba**
1. Added popup window
2. Added project cards dynamically using Javascript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -19,6 +20,171 @@ function closer() {
...
+    <button class="snapshot-btn1" onclick="javascript:window.open('${popup[0].live}', '_blank');">See Live <img src="/Images/live.png" class="btnpng"></button>
+    <button class="snapshot-btn" onclick="javascript:window.open('${popup[0].source}', '_blank');">See Source <img src="/Images/git.png" class="btnpng"></button>
+    </div>
+
+    </div>
+
+</div>`;
+  midpart.appendChild(modalContainer);
+}
+popupWindow();
```
Comment (rotimiazeez) [inline script.js]: - Kindly make sure the images on your pop-up, see live and see source button respectively are rendering correcting 😉 as seen on the original Figma design 👍🏽
Your Implementation | Figma Design
:---:|:---:
screenshot (15)
| 
screenshot (16)
screenshot (17)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👋🏽, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit messages 👍
- Descriptive PR message
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tafaramafemba** [comment]
screenshot2

screenshot1
Hi @rotimiazeez !! I have checked the review. On my end the images are showing so I am not too sure what to do to ensure they show on yours. Please help me out.

**Comment by rotimiazeez** [comment]
Hi @tafaramafemba,
I think all you need to do is render the link as ./images/....

**Comment by tafaramafemba** [comment]
> Hi @tafaramafemba, I think all you need to do is render the link as ./images/....

@rotimiazeez Done. Let me know if that works.

**Comment by rotimiazeez** [comment]
It's good now 😉 
screenshot (18)

**Review by gemmen29 — APPROVED** [review]
Hi @tafaramafemba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**